code/modules/cargo/gondolapod.dm

```diff
@@ -65,11 +65,13 @@
 
 /mob/living/simple_animal/pet/gondola/gondolapod/setOpened()
 	opened = TRUE
+	SET_PLANE_IMPLICIT(src, GAME_PLANE)
```

Comment:
looking for review here, I am not proficient with planes but this did the job I wanted it to.

Reply:
@LemonInTheDark

Reply:
looks good. it pulls the z level from the pod, and offsets the plane based off that. thanks!